Gemfile

```
@@ -3,7 +3,7 @@
 source 'https://rubygems.org'
 ruby File.read('.ruby-version')
 
-gem 'rails', '~> 6.1.4'
+gem 'rails', '~> 7.0'
```

This'll allow 7.1+ which we don't want (at least, not to just "happen" as part of someone doing `bundle update rails` for a security patch):

```suggestion
gem 'rails', '~> 7.0.0'
```

---

Nice catch!